core/matrix/dense.cpp

```diff
@@ -814,15 +881,14 @@ std::unique_ptr<LinOp> Dense<ValueType>::inverse_column_permute(
 
 template <typename ValueType>
 std::unique_ptr<LinOp> Dense<ValueType>::inverse_column_permute(
-    const Array<int64> *inverse_permutation_indices) const
+    const Array<int64> *permutation_indices) const
```

Comment:
The permutation_indices will be inverted internally, but the parameter name right now implies that it would take the inverse permutation as input, which is misleading.